README.md

```diff
@@ -50,22 +50,32 @@ The following environment variables can be used to configure how the framework e
 
 #### Run Examples on the Web (`wasm32-unknown-unknown`)
 
-Running on the web is still work-in-progress. You may need to enable experimental flags on your browser. Check browser implementation status on [webgpu.io](https://webgpu.io). Notably, `wgpu-rs` is often ahead in catching up with upstream WebGPU API changes. We keep the `gecko` branch pointing to the code that should work on latest Firefox.
+##### WebGL and WebGPU
```

Comment:
Maybe the section could just be "Web" and give an 1-2 sentence overview of how/why two possible backends exist, with separate subsections for "WebGPU" and "WebGL"